src/Services/DatabaseTools/PHPCRDatabaseTool.php

```diff
@@ -1,104 +0,0 @@
-<?php
```

Comment:
why remove this?

Reply:
It broke tests because it uses a command from the old bundle: https://travis-ci.com/liip/LiipTestFixturesBundle/jobs/185829031

I wanted a :heavy_check_mark: from CI so I removed it.

I'll try to restore this.

Reply:
I would just copy the runCommand code into the test for now

Reply:
I used Symfony's CommandTester and it worked.